website/i18n/en.json

```diff
@@ -177,10 +177,6 @@
         "title": "Treasury",
         "sidebar_label": "Treasury"
       },
-      "learn-UI": {
-        "title": "Polkadot UI",
-        "sidebar_label": "Polkadot UI"
-      },
```

Comment:
Why did you get rid of this?

Reply:
Accidental. That page was removed before because it seemingly wasn't linked in any sidebar (it was causing problems in the link checker), so I cleaned up the JSON here too. Was it supposed to stick around? I thought it was orphaned for a reason (outdatedness).

Reply:
No, I didn't know it was already removed and it didn't seem relevant to the rest of the PR, so I asked.  May as well just get rid of it now.